Update database-import-export-hang.md with auto-cancellation note (#8129)
MicrosoftDocs/sql-docs: azure-sql/database/database-import-export-hang.md

Additionally, as the Import/Export service performs a logical backup of the database the time to complete is more dependent on the number of objects in the database than a traditional physical database backup.

> [!NOTE]

> > After resources are assigned and a request begins to be processed, the service will automatically cancel the request after two days.

Reviewer: ```suggestion
> After resources are assigned and a request starts to process, the service automatically cancels the request after two days.
```

## Things to consider when you export or import a database

> * All the methods discussed in this article use up the Database Transaction Unit (DTU) or CPU quota, which causes throttling by the Azure SQL Database service. You can [view the DTU stats for the database on the Azure portal](./monitor-tune-overview.md#azure-sql-database-and-azure-sql-managed-instance-resource-monitoring). If the database has reached its resource limits, [upgrade the service tier](./scale-resources.md) to add more resources.

Reviewer: ```suggestion
* All the methods discussed in this article consume Database Transaction Units (DTUs) or CPU quota, which causes throttling by the Azure SQL Database service. You can [view the DTU stats for the database on the Azure portal](./monitor-tune-overview.md#azure-sql-database-and-azure-sql-managed-instance-resource-monitoring). If the database has reached its resource limits, [upgrade the service tier](./scale-resources.md) to add more resources.
```
